conference/src/main/java/com/kyriba/conference/management/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,62 @@
+package com.kyriba.conference.management;
+
+
+import com.kyriba.conference.management.domain.exception.EntityNotFound;
+import com.kyriba.conference.management.domain.exception.InvalidPresentationTime;
+import com.kyriba.conference.management.domain.exception.LinkedEntityNotFound;
+import com.kyriba.conference.management.domain.exception.TimeslotIsBooked;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+
+
+@ControllerAdvice
+public class GlobalExceptionHandler
```

Comment:
It is better to declare error handling and transformation to HTTP in appropriate controllers if errors are not global

Reply:
done